docs/programming-oak.md

```diff
@@ -477,6 +477,31 @@ handle for the _write_ half of a different channel, which is then used for
 responses &ndash; so the new Node has a way of _sending_ externally, as well as
 receiving.
 
+If two nodes communicate using Protocol Buffers, channel handles can be shared
+by encoding them in a message:
```

Comment:
To me this feels a bit like explaining implementation details, is that useful for people that want to get started writing Oak nodes?
It might still be nice to document the internals a bit, though I don't think there is such a thing as an 'Oak internals' page?

Reply:
Actually, I think you are right that we should mention that this is an extension provided by the SDK and flags a field as a handle. I'll try and include that 😄.